modules/nw-ipfailover-configuration.adoc

```diff
@@ -0,0 +1,182 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ipfailover.adoc
+
+[id="nw-ipfailover-configuration_{context}"]
+= Configuring IP failover
+
+As a cluster administrator, you can configure IP failover on an entire cluster, or on a subset of nodes, as defined by the label selector. You can also configure multiple IP failover deployment configurations in your cluster, where each one is independent of the others.
+
+The IP failover deployment configuration ensures that a failover pod runs on each of the nodes matching the constraints or the label used.
+
+This pod runs Keepalived, which can monitor an endpoint and use Virtual Router Redundancy Protocol (VRRP) to fail over the virtual IP (VIP) from one node to another if the first node cannot reach the service or endpoint.
+
+For production use, set a `selector` that selects at least two nodes, and set `replicas` equal to the number of selected nodes.
+
+.Prerequisites
+
+* You are logged in to the cluster with a user with `cluster-admin` privileges.
+* You created a pull secret.
+
+.Procedure
+
+//. Create an {product-title} pull secret
+//+
+. Create an IP failover service account:
++
+[source,terminal]
+----
+$ oc create sa ipfailover
+----
++
+. Update security context constraints (SCC) for `hostNetwork`:
++
+[source,terminal]
+----
+$ oc adm policy add-scc-to-user privileged -z ipfailover
+$ oc adm policy add-scc-to-user hostnetwork -z ipfailover
+----
++
+. Create a deployment YAML file to configure IP failover:
++
+.Example deployment YAML for IP failover configuration
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ipfailover-keepalived <1>
+  labels:
+    ipfailover: hello-openshift
+spec:
+  strategy:
+    type: Recreate
+  replicas: 2
+  selector:
+    matchLabels:
+      ipfailover: hello-openshift
+  triggers:
+    type: ConfigChange
```

Comment:
Are triggers only for DeploymentConfigs?

Reply:
I'm not sure. I'll default to @rfredette on that.

Reply:
Yes, triggers are only supported in DeploymentConfigs. I'm surprised I didn't get any warnings from oc when trying to define one here, but I guess it just ignored the unexpected field. It can be removed.

Reply:
Thank you. I didn't see this update before I merged, so I'll open a follow-up PR to remove that line.